Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents.py

```diff
@@ -921,7 +944,7 @@ def main():
                                              exact_match_fields, display_fields, from_date, to_date, confidence,
                                              max_incidents, aggregate, limit, incident_id,
                                              ])
-    demisto.debug(f"Starting,\n{fields_values=}")
+    demisto.debug(f"DBS: Starting,\n{fields_values=}")
```

Comment:
can we print each field separately instead of using build_message_of_values()?
e.g. {similar_text_field=}, {similar_json_field=}, etc.
this debug log is not good, we need to guess for each field value what is its type